spec/intelligent_foods/resources/menu_spec.rb

```diff
@@ -24,4 +24,79 @@
       end
     end
   end
+
+  describe ".find" do
+    it "returns the menu" do
+      menu_id = "2023-01-01"
+      body = build_menu_response(menu_id: menu_id)
+      response = build_response(body: body)
+      stub_api_response response: response
+
+      result = IntelligentFoods::Menu.find(menu_id)
+
+      expect(result.id).to eq(menu_id)
+    end
+
+    it "returns an instance of IntelligentFoods::Menu" do
+      menu_id = "2023-01-01"
+      body = build_menu_response(menu_id: menu_id)
+      response = build_response(body: body)
+      stub_api_response response: response
+
+      result = IntelligentFoods::Menu.find(menu_id)
+
+      expect(result).to be_an_instance_of IntelligentFoods::Menu
+    end
+
+    it "returns MenuItem of proper type" do
+      menu_id = "2023-01-01"
+      menu_items = stubbed_menu_items
+      body = build_menu_response(menu_id: menu_id, menu_items: menu_items)
+      response = build_response(body: body)
+      stub_api_response response: response
+      menu = IntelligentFoods::Menu.find(menu_id)
+
+      result = menu.items.first
+
+      expect(result).to be_an_instance_of IntelligentFoods::MenuItem
+    end
+
+    it "assigns correct number of items" do
+      menu_id = "2023-01-01"
+      menu_items = stubbed_menu_items
+      expected_items_count = menu_items.count
+      body = build_menu_response(menu_id: menu_id, menu_items: menu_items)
+      response = build_response(body: body)
+      stub_api_response response: response
+      menu = IntelligentFoods::Menu.find(menu_id)
+
+      result = menu.items.count
```

Comment:
```suggestion
      result = menu.items.size
```

prefer `size` over `count`